### PR TITLE
[MISC] Ability to change Xenia process priority

### PR DESCRIPTION
**Pull request by Xphalnos**
I added the feature that the Xenia process is **Above Normal** instead of **Normal**. This can surely improve performance.

### CODE REVIEW

File: src/xenia/emulator.cc [hunk 1]
```diff
@@ -226,6 +226,11 @@ X_STATUS Emulator::Setup(
     backend.reset(new xe::cpu::backend::NullBackend());
   }
 
+#if XE_PLATFORM_WIN32
```
Comment (Gliniak) [inline src/xenia/emulator.cc]: This (or imGui part) should be different PR. You can and should put it in: ``int WINAPI wWinMain`` in file: windowed_app_main_win.cc
I would also allow user to select priority class in config.

My only concern is fact that games internally use SetThreadPriority and they can even check it and that might interfere with games stability

**Comment by Xphalnos** [comment]
It's ok I did what you asked. I tested on GTA IV and everything seems to work.

File: src/xenia/ui/windowed_app_main_win.cc [hunk 2]
```diff
@@ -351,6 +351,9 @@ int WINAPI wWinMain(HINSTANCE hinstance, HINSTANCE hinstance_prev,
   g_xenia_exe_base = reinterpret_cast<uintptr_t>(hinstance);
   g_xenia_exe_size = modinfo.SizeOfImage;
 
+  // Set "Above Normal" priority.
+  SetPriorityClass(GetCurrentProcess(), ABOVE_NORMAL_PRIORITY_CLASS);
```
Comment (Gliniak) [inline src/xenia/ui/windowed_app_main_win.cc]: Still I would leave it to user to select priority class by some config option if possible. For example via some config option that allows to select:
0 - Normal
1 - Above Normal
2 - High

Problem is that I don't know if config is parsed at this point. I kinda doubt it, but it should be checked

**Comment by Xphalnos** [comment]
I still don't quite understand Xenia's configuration system.
I added a temporary file `test.h` because I don't know where to put `DECLARE_int32(windows_priority)`.
I also have a problem because the `windows_priority` is always 0 (probably due to the configuration problem).
Maybe @Gliniak can help me?

**Comment by Gliniak** [comment]
I updated this PR

**Comment by Xphalnos** [comment]
Thanks Gliniak! your work is so much cleaner ;)
